### DISCUSSION TITLE
Run trivy for k8s cluster  for compliance

### DISCUSSION THREAD

**Discussion by senemamourr** [aquasecurity/trivy#4368] [Ideas]
I'm looking to use the trivy cli to  check compliance on my cluster.
Due to psa issue, the node-controller cannot be launch.
is it  possible do specifie namespace where the  scan job (node-controller) will run ?
thank you.

**Comment by chen-keinan** [comment]
@senemamourr the `node-controller` is running on `trivy-temp` namespace

**Comment by senemamourr** [comment]
Thank you @chen-keinan for your reply.
Please is it possible to set the namespace where it gonna run ?

**Comment by chen-keinan** [comment]
@senemamourr sure , I'll check the possibility for adding this capability and update you

**Reply by knqyf263** [reply]
@chen-keinan Any updates?

**Reply by chen-keinan** [reply]
@knqyf263 can be done , I'll add support within next release
issue #4371

**Reply by knqyf263** [reply]
@senemamourr Please watch #4371
